src/js/contexts/AnnounceContext/AnnounceContext.js

```diff
@@ -13,6 +13,12 @@ const createAnnouncer = () => {
 
   return announcer;
 };
+/*
+Prime the live region so that it's registered with the browser
+when we first call the announce function.
+See https://tetralogical.com/blog/2024/05/01/why-are-my-live-regions-not-working/
+*/
+createAnnouncer();
```

Comment:
This should fix a bug where the first data bar isn't announced, when you first press the right arrow, because the live region hasn't been registered with the browser yet.

Reply:
I’m inclined to revert this commit (and the snapshot tests) and focus this PR just on the `DataChart` component.